### PR TITLE
Fix output of API for overTimeData if there are 0 blocked queries

### PR DESCRIPTION
**Pull request by DL6ER**
**By submitting this pull request, I confirm the following (please check boxes, eg [X] - no spaces) _Failure to fill the template will close your PR_:**

***Please submit all pull requests against the `development` branch. Failure to do so will delay or deny your request***

- [X] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md).
- [X] I have checked that [another pull request](https://github.com/pi-hole/pi-hole/pulls) for this purpose does not exist.
- [X] I have considered, and confirmed that this submission will be valuable to others.
- [X] I accept that this submission may not be used, and the pull request closed at the will of the maintainer.
- [X] I give this submission freely, and claim no ownership to its content.

**How familiar are you with the codebase?:** 10

---

Provide a minimal valid array if there have are no blocked queries at all. Currently, the output of the API is inconsistent in this case preventing the main graph from showing any data at all.

_This template was created based on the work of [`udemy-dl`](https://github.com/nishad/udemy-dl/blob/master/LICENSE)._

### CODE REVIEW

File: data.php [hunk 1]
```diff
@@ -68,6 +76,16 @@ function getOverTimeData10mins() {
...
         $ads_over_time = overTime10mins($ads_blocked);
+
+        // Provide a minimal valid array if there have are no blocked
+        // queries at all. Otherwise the output of the API is inconsistent.
+        if(count($ads_blocked) == 0)
+        {
+            $ads_over_time = [1 => 0];
+        }
+
+        print_r($domains_over_time);
```
Comment (Mcat12) [inline data.php]: There's some extra debug output.

**Comment by Mcat12** [comment]
What kind of inconsistencies are you seeing? I haven't seen any.

**Comment by Mcat12** [comment]
Might be because I don't have enough data to show much on the graph.

**Comment by DL6ER** [comment]
If we have 0 blocked, we get
```
pi.hole/admin/api.php?overTimeData10mins
{"domains_over_time":{"4":"1","9":"3","91":"2","93":"2","99":"2"},"ads_over_time":[]}
```
which prevents the main graph from being rendered correctly.

With this PR we get
```
pi.hole/admin/api.php?overTimeData10mins
{"1":"0","2":"0","3":"0","4":"1","5":"0","6":"0","7":"0","8":"0","9":"3","10":"0","11":"0","12":"0","13":"0","14":"0","15":"0","16":"0","17":"0","18":"0","19":"0","20":"0","21":"0","22":"0","23":"0","24":"0","25":"0","26":"0","27":"0","28":"0","29":"0","30":"0","31":"0","32":"0","33":"0","34":"0","35":"0","36":"0","37":"0","38":"0","39":"0","40":"0","41":"0","42":"0","43":"0","44":"0","45":"0","46":"0","47":"0","48":"0","49":"0","50":"0","51":"0","52":"0","53":"0","54":"0","55":"0","56":"0","57":"0","58":"0","59":"0","60":"0","61":"0","62":"0","63":"0","64":"0","65":"0","66":"0","67":"0","68":"0","69":"0","70":"0","71":"0","72":"0","73":"0","74":"0","75":"0","76":"0","77":"0","78":"0","79":"0","80":"0","81":"0","82":"0","83":"0","84":"0","85":"0","86":"0","87":"0","88":"0","89":"0","90":"0","91":"2","92":"0","93":"2","94":"0","95":"0","96":"0","97":"0","98":"0","99":"2"},"ads_over_time":{"1":"0","2":"0","3":"0","4":"0","5":"0","6":"0","7":"0","8":"0","9":"0","10":"0","11":"0","12":"0","13":"0","14":"0","15":"0","16":"0","17":"0","18":"0","19":"0","20":"0","21":"0","22":"0","23":"0","24":"0","25":"0","26":"0","27":"0","28":"0","29":"0","30":"0","31":"0","32":"0","33":"0","34":"0","35":"0","36":"0","37":"0","38":"0","39":"0","40":"0","41":"0","42":"0","43":"0","44":"0","45":"0","46":"0","47":"0","48":"0","49":"0","50":"0","51":"0","52":"0","53":"0","54":"0","55":"0","56":"0","57":"0","58":"0","59":"0","60":"0","61":"0","62":"0","63":"0","64":"0","65":"0","66":"0","67":"0","68":"0","69":"0","70":"0","71":"0","72":"0","73":"0","74":"0","75":"0","76":"0","77":"0","78":"0","79":"0","80":"0","81":"0","82":"0","83":"0","84":"0","85":"0","86":"0","87":"0","88":"0","89":"0","90":"0","91":"0","92":"0","93":"0","94":"0","95":"0","96":"0","97":"0","98":"0","99":"0"}}
```

**Comment by Mcat12** [comment]
Approved

**Pull request merged**